css-view-transitions-1/Overview.bs

```diff
@@ -344,14 +692,24 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 
 		Note: Additional styles in the [=document/view transition style sheet=] added to animate these pseudo-elements are detailed in [=setup transition pseudo-elements=] and [=update pseudo-element styles=].
 
-		Note: This is only ever a [=tree/child=] of a ''::view-transition-image-pair()'', and never has any [=tree/children=].
-
 	: <dfn>::view-transition-new( <<pt-name-selector>> )</dfn>
 	:: Identical to ''::view-transition-old()'',
-		except it deals with the new element instead.
+		except the following styles added to the [=HTML user agent style sheet=]:
 
-	Note: The precise tree structure, and in particular the order of sibling pseudo-elements,
-	is defined in the [=setup transition pseudo-elements=] algorithm.
+		```css
+		html::view-transition-old(*) {
+			position: absolute;
+			inset-block-start: 0;
+			inline-size: 100%;
+			block-size: auto;
+
+			animation-name: -ua-view-transition-fade-in;
+			animation-duration: inherit;
+			animation-fill-mode: inherit;
+		}
+		```
```

Comment:
Felt like it was worth specifying this in full, especially due to `-ua-view-transition-fade-in`.